eth/backend.go

```diff
@@ -340,7 +353,19 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		}
 		eth.historicalRPCService = client
 	}
-	*/
+
+	// [Kroma: ZKT to MPT]
+	// Start the background state migrator
+	if eth.blockchain.Config().Zktrie && eth.blockchain.Config().KromaMPTTime != nil && !config.DisableMPTMigration {
+		head := eth.BlockChain().CurrentBlock()
+		if !eth.blockchain.Config().IsKromaMPT(head.Time) {
```

Comment:
Don't we have to check KromaMPTTime != 0 here for cases that we wanna spin up geth with MPT?

Reply:
If KromaMPTTime is 0, IsKromaMPT() will be true and the migrator will not start, so don't need to check it.